assets/script/script.js

```diff
@@ -8,26 +11,91 @@ fetch("https://api.football-data.org/v2/competitions/PL/teams", {
     return response.json();
   })
   .then(function (data) {
-    footballData = data;
+    teamData = data;
   })
   .then(renderCrests)
-  .then(renderTeamInfo);
+  .then(fillTagsArray);
 
+// get data for league standings
+fetch("https://api.football-data.org/v2/competitions/PL/standings", {
+  headers: { "X-Auth-Token": "b7bac95afb44489a83d8eb77fc894151" },
+})
+  .then(function (response) {
+    return response.json();
+  })
+  .then(function (data) {
+    standings = data;
+  });
 
 // adds all crests to the welcome page
 function renderCrests() {
-  for (i = 0; i < footballData.teams.length; i++) {
+  for (let i = 0; i < teamData.teams.length; i++) {
     $("#crestContainer").append(
       $(
-        `<img class='crests' alt='Crest for ${footballData.teams[i].name} 
-        'title='${footballData.teams[i].name}' src=${footballData.teams[i].crestUrl}>`
+        `<img class='crests' alt='Crest for ${teamData.teams[i].name} 
+        'title='${teamData.teams[i].name}' data-id="${teamData.teams[i].id}" src=${teamData.teams[i].crestUrl}>`
       )
     );
   }
 }
 
-// testing function for add individual crest to page
-function renderTeamInfo() {
-    $('#teamCrest').attr('src', `${footballData.teams[0].crestUrl}`)
-    $('#teamName').text(`${footballData.teams[0].name}`)
+// function for adding team info on search button
+let selectedTeamId = "";
+function renderTeamInfo(e) {
+  e.preventDefault();
+  renderTeam()
+  renderLeagueStats();
+  $("#teamSearch").val("");
+}
+
+// event listener for search button
+$("#searchButton").click(renderTeamInfo);
+
+// function for filling tags array
+function fillTagsArray() {
+  for (let i = 0; i < teamData.teams.length; i++) {
+    tagsArray.push(teamData.teams[i].name);
+  }
+}
+
+$(function () {
+  $("#teamSearch").autocomplete({
+    source: tagsArray,
+  });
+});
+
+
+function renderTeam(){
+    for (let i = 0; i < teamData.teams.length; i++) {
```

Comment:
There is a `foreach` in JS: https://www.w3schools.com/jsref/jsref_forEach.asp

That might make this block of code a little neater.

Reply:
Or just define `const team = teamData.teams[i]` at the top of your for loop.